include/common.h

```diff
@@ -63,6 +63,14 @@
 #define HC_API_CALL
 #endif
 
+#if defined (__GNUC__)
+#define HC_ALIGN(x) __attribute__((aligned(x)))
+#elif defined (_MSC_VER)
+#define HC_ALIGN(x) __declspec(align(x))
+#else
+#define HC_ALIGN(x)
```

Comment:
Macros are fine, we can leave them. But do we really need them and why? Btw. I don't know if _MSC_VER is really supported, never tried. But will it work on clang?

Reply:
> Macros are fine, we can leave them. But do we really need them and why? Btw. I don't know if _MSC_VER is really supported, never tried. But will it work on clang?

I have used this same macro before with GCC, clang, Intel and MSVC.